api_by_features/battery.json

```diff
@@ -0,0 +1,45 @@
+{
+    "Baterry": {
+    	"description": "Ability to read level and mode of charge of baterry.",
+        "browser": {
+        	"API": "Baterry Status API",
+        	"spec_url": "http://www.w3.org/TR/battery-status/",
+        	"working group": ""
```

Comment:
as I alluded to the other day during the Skype call, I think we should use the same structure for "browser" and for the other platforms, i.e. "name" instead of "API", and use "url" instead of "spec_url" (and change "docs" to "url" in the other platforms section). We should also avoid having "working group" in there since that's data we can get from elsewhere.